website/models.py

```diff
@@ -47,6 +47,7 @@ def generate_uuid_for_company(apps, schema_editor):
 
 
 class Company(models.Model):
+    company_id = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
```

Comment:
Maybe we should create an [abstract base model](https://docs.djangoproject.com/en/5.0/topics/db/models/#abstract-base-classes) with `uuid` fields and use it for inheritance when we need to have uuid field (the name would be the same for all models)?

Reply:
@arkid15r sir, I update the code